parser-typechecker/src/Unison/Runtime/Builtin.hs

```diff
@@ -866,24 +866,39 @@ set'buffering instr
           $ outIoFailUnit s1 s2 s3 u f r
   (handle,bmode,tag,n,w,s1,s2,s3,u,f,r) = fresh11
 
-get'buffering'output :: forall v. Var v => v -> v -> v -> ANormal v
-get'buffering'output bu n w =
-  TMatch bu . MatchSum  $ mapFromList
-  [ no'buf --> [] --> TCon Ty.bufferModeRef no'buf []
-  , line'buf --> [] --> TCon Ty.bufferModeRef line'buf []
-  , block'buf --> [] --> TCon Ty.bufferModeRef block'buf []
-  , sblock'buf --> [UN] -->
-        TAbs w
-      . TLetD n BX (TCon Ty.natRef 0 [w])
-      $ TCon Ty.bufferModeRef sblock'buf [n]
-  ]
+get'buffering'output :: forall v. Var v => v -> v -> v -> v -> v -> v -> ANormal v
```

Comment:
I worked with @dolio  to fix this;
we also attempted to abstract a helper for fmap'ing over an Either like this, but unfortunately it's not easy to do so.